src/components/ReportActionItem/IOUAction.js

```diff
@@ -43,29 +63,52 @@ const IOUAction = (props) => {
     const launchDetailsModal = () => {
         Navigation.navigate(ROUTES.getIouDetailsRoute(props.chatReportID, props.action.originalMessage.IOUReportID));
     };
+
+    const shouldShowIOUPreview = (
+        props.isMostRecentIOUReportAction && Boolean(props.action.originalMessage.IOUReportID)
+    ) || props.action.originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY;
+
+    let shouldShowPendingConversionMessage = false;
```

Comment:
Can't we just assign this with the return value from the `isIOUReportPendingCurrencyConversion` method because essentially it does the same thing, right?

Reply:
We also need to check if we are in the IOUPreview action so that we avoid having to run this code unnecessarily in other actions.

Reply:
I am thinking if we ever show a pending message then it has to be for the IOUPreview, right? So can't we just do 

```js
shouldShowPendingConversionMessage={IOUUtils.isIOUReportPendingCurrencyConversion(props.reportActions, props.iouReport);}
```
because only when IOUPreview will be rendered this expression will get evaluated and I can also see in the `isIOUReportPendingCurrencyConversion` we are already checking for pending actions to the `if` block seems unnecessary to me.

What do you think about it?

Reply:
I'm not sure. This will make `isIOUReportPendingCurrencyConversion` even when we're online, and even when we're showing an IOUPreview of an action of type `pay` (the one with a message `You paid user@email.com`. I added that check so that we only that function if it's the last IOUPreview of a chat that has an outstanding IOU and we're offline. Let me know what you think.